lucene/core/src/java/org/apache/lucene/index/DirectoryReader.java

```diff
@@ -104,6 +104,23 @@ public static DirectoryReader open(final IndexCommit commit) throws IOException
     return StandardDirectoryReader.open(commit.getDirectory(), commit);
   }
 
+  /**
+   * Expert: returns an IndexReader reading the index in the given {@link IndexCommit}. This method
+   * allows to open indices that were created wih a Lucene version older than N-1 provided that all
+   * all codecs for this index are available in the classpath and the segment file format used was
+   * created with Lucene 7 or older. Users of this API must be aware that Lucene doesn't guarantee
```

Comment:
Instead of "Lucene 7" should we reference the `minSupportedMajorVersion` parameter here? Or .. is it  `Version.MIN_SUPPORTED_MAJOR`?

Reply:
this is due to the fact that the segments info format only supports 7.0 and upwards

Reply:
The javadoc should read 'Lucene 7 or newer' I think?

Reply:
👍